package.json

```diff
@@ -46,7 +44,8 @@
     "ts-jest": "29.2.4",
     "ts-loader": "9.5.1",
     "tsx": "4.17.0",
-    "typescript": "5.5.4",
+    "typescript": "^5.5.4",
```

Comment:
pls make all version be exact (without `^` symbol). 

`npm config set save-exact=true` and reinstall these packages.